base/deprecated.jl

```diff
@@ -209,7 +209,7 @@ for deprecatedfunc in [:isprime, :primes, :primesmask, :factor]
     @eval begin
         $deprecatedfunc(args...) = error(string($deprecatedfunc, args,
             " has been moved to the package Primes.jl.\n",
-            "Run Pkg.add(\"Primes\") to install Primes on Julia v0.5-"))
+            "Run Pkg.add(\"Primes\") to install Primes on Julia v0.5- and then run `using Primes`."))
```

Comment:
`0.5-` means 0.5 prerelease. I did not mean to imply that `v0.5-` meant "v0.5 or later", but rather that "v0.5 or later" is better than `0.5-`. Your proposed migration message is fine.